database/migrations/2023_04_26_104633_create_landing_pages_tables_from_pages_table.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class CreateLandingPagesTablesFromPagesTable extends Migration
+{
+    public function up()
+    {
+        // Create landing_pages tables
+
+        DB::statement('CREATE TABLE landing_pages AS TABLE pages');
+        DB::statement('CREATE SEQUENCE landing_pages_id_seq AS integer');
```

Comment:
You'll need an alter column here as well to apply this sequence to the `landing_pages` table:

```
DB::statement("ALTER TABLE landing_pages ALTER COLUMN id SET DEFAULT nextval('landing_page_id_seq'::regclass)");
```